source/services/circle-ci.rst

```diff
@@ -0,0 +1,36 @@
+Circle CI
+=========
+
+The project template now has a Circle CI template that you can use for your project. Circle CI's
+benefit over Travis CI is that it works for private non-Caktus repos. Once you have
+completed the initial project setup (i.e. the items in README.rst), do the following:
+
+1. If you don't have one already, `create a Circle CI account <https://circleci.com/>`_, being sure
+   to link it to your Caktus github account.
```

Comment:
How does one `make sure to link it...`?  I know with some online services it's not obvious how to do that in a way that'll end up working for us.